src/main/java/org/sert2521/gamename/Drivetrain.kt

```diff
@@ -10,7 +10,6 @@ import edu.wpi.first.wpilibj.command.Subsystem
  * This is the Drivetrain subsystem, which is in charge of controlling the speeds of the individual
  * talons and converting joystick input to movement speeds.
  */
-
```

Comment:
I'm gunna do this in another commit so we don't change files unnecessarily in this PR.